ingestion/src/metadata/profiler/orm/functions/median.py

```diff
@@ -116,10 +116,28 @@ def _(elements, compiler, **kwargs):
 
 @compiles(MedianFn, Dialects.MySQL)
 def _(elements, compiler, **kwargs):  # pylint: disable=unused-argument
-    """Median computation for MySQL currently not supported
-    Needs to be tackled in https://github.com/open-metadata/OpenMetadata/issues/6340
-    """
-    return "NULL"
+    """Median computation for MySQL"""
+    col = compiler.process(elements.clauses.clauses[0])
+    table = elements.clauses.clauses[1].value
+    percentile = elements.clauses.clauses[2].value
+
+    return """
+    (SELECT
+        {col}
+    FROM (
+        SELECT
+            t.{col}, 
+            @row_num := @row_num + 1 AS row_num
+        FROM 
+            {table} t,
+            (SELECT @row_num:=0) counter 
```

Comment:
@schlameel just wondering why we are doing this cross join here?

Reply:
It is really just setting an initial value for `@row_num. Changed.